src/main/java/com/esri/samples/localserver/local_server_map_image_layer/LocalServerMapImageLayerSample.java

```diff
@@ -75,7 +75,7 @@ public void start(Stage stage) throws Exception {
       server.addStatusChangedListener(status -> {
         if (status.getNewStatus() == LocalServerStatus.STARTED) {
           // start map image service
-          String mapServiceURL = new File("./samples-data/local_server/RelationshipID.mpk").getAbsolutePath();
+          String mapServiceURL = new File("./samples-data/local_server/USCitiesStates.mpk").getAbsolutePath();

```

Comment:
I'll merge this with the change. Let me know if you're sure we can use the mpk and if the sample in master needs to be updated and I'll make those changes.